### ISSUE TITLE
Enabling Autocomplete on Cell

### ISSUE THREAD

**Issue by Bshara23** [datalayer/jupyter-ui#82]
### Problem
Trying to enable auto completion but can't find any resource or example on how to do so
I'm using this example project: https://github.com/datalayer/jupyter-ui/tree/main/examples/cra 

### Suggested Improvement
Please provide an example or documentation on how to enable auto completion
If it's enabled by default, can you provide suggestions on why it's not working

**Comment by echarles** [comment]
The notebook has autocomplete support

<img width="211" alt="Screenshot 2023-07-30 at 20 13 26" src="https://github.com/datalayer/jupyter-ui/assets/226720/eb9c7a57-c4ad-4b7e-9f7c-b0fc5381414d">

<img width="282" alt="Screenshot 2023-07-30 at 20 13 44" src="https://github.com/datalayer/jupyter-ui/assets/226720/cf4aab9b-b0b7-40aa-b7d6-54b6a8400dbd">


The cell does not have it yet, will work on that soon.

BTW this is also linked to https://github.com/datalayer/jupyter-ui/issues/33

**Comment by echarles** [comment]
@Bshara23 is this still an issue for you? If not, let's close this.

**Comment by Bshara23** [comment]
I thought it would be better to keep it open until the autocomplete feature (from the cell) is supported so other users won't open a similar issue.
Will close it

**Comment by echarles** [comment]
> until the autocomplete feature (from the cell) is supported so other users won't open a similar issue.

You are right. Autocomplete is only supported on `Notebook`, not yet on `Cell`. Reopening (also updating the title).

**Comment by BrandonEscamilla** [comment]
Hi @echarles, could you please provide instructions on how to enable autocomplete in Jupyter Notebook? Continuous autocomplete isn't supported in my case. Additionally, I would appreciate instructions on how to show the documentation panel. Thank you in advance!

**Comment by echarles** [comment]
It is working, but you need to run at least one cell before it becomes available. You just need to press TAB key.

![Untitled6](https://github.com/datalayer/jupyter-ui/assets/226720/bfe4f7bf-1057-435d-bd09-cec214291423)

**Comment by BrandonEscamilla** [comment]
Hey @echarles, I noticed that if you enable "autocompletion" and "show documentation panel" in jupyterlab, the behavior is to show suggestions as you type along (with no Tab) with the corresponding documentation string. Would it be possible to implement this functionality, or would the react version will only support the Tab key for completion without displaying doc strings? Thanks in advance!

![CleanShot 2023-09-27 at 09 46 58](https://github.com/datalayer/jupyter-ui/assets/21966716/04704c76-d158-4385-b7d7-7bd39a95179f)
![CleanShot 2023-09-27 at 09 46 05](https://github.com/datalayer/jupyter-ui/assets/21966716/84d5e0f5-db41-4a95-bf85-f2a678d929b6)

**Comment by echarles** [comment]
@BrandonEscamilla Thx for sharing more details on that. To support all the options (that one, or any one provided by JupyterLab), Jupyter-UI (this library) is moving towards a JupyterLabApp component (see https://github.com/datalayer/jupyter-ui/issues/67 and https://github.com/datalayer/jupyter-ui/issues/106#issuecomment-1736955504).

I guess that feature will need to wait on a new `NotebookApp` component.
